### DISCUSSION TITLE
communication between mastodon servers

### DISCUSSION THREAD

**Discussion by 13antlers** [mastodon/mastodon#18243] [Q&A]
can mastodon accounts/users communicate between servers, i.e., share posts/follows with users on other mastodon servers? trying to get folks to migrate over from twitter, but want to let them know how it works...

**Comment by ClearlyClaire** [comment]
yes, this is one of the main features of Mastodon, it federates with other Mastodon instances as well as other compatible software

**Comment by 13antlers** [comment]
ok great! for some reason, i can’t yet see the posts of people i follow on other servers, and vice versa… how to fix?

**Reply by ClearlyClaire** [reply]
This could be a bug, a deployment issue, a misunderstanding, or servers blocking each other. I'll need more information to be able to help here!

Are new posts from someone you already follow not appearing? Posts from a remote account are not back-filled on your server, but every new post made when you are following them should appear.

When you look at the their profile from your server, is it telling you that you are following them? Or maybe that you have sent a follow request?

Are you using your own server, or have you joined an existing one? What about the people you're trying to follow?

**Comment by 13antlers** [comment]
ah ok, it’s probably the back-fill issue. will check this out. thx for your help!
